tests/acceptance/features/bootstrap/NotificationsCoreContext.php

```diff
@@ -140,7 +153,10 @@ public function userNumNotifications($user, $numNotifications, $missingLast) {
 				$now[] = $this->getDeletedNotification();
 			}
 
-			Assert::assertEquals($previousNotificationIds, $now);
+			Assert::assertEquals(
+				$previousNotificationIds,
+				$now
```

Comment:
```suggestion
				$now,
				"The actual notifications, {$missingLast}, are not what was expected. {$numNotifications} were expected. See the differences below."
```
Maybe a message like this will work? `assertEquals` will print a diff of the 2 arrays.